docs/go-workers.md

```diff
@@ -4,9 +4,14 @@ title: Workers in Go
 sidebar_label: Workers
 ---
 
-A worker or _worker service_ is a service that hosts the Workflow and Activity implementations. The worker polls the _Temporal service_ for tasks, performs those tasks, and communicates task execution results back to the _Temporal service_. Worker services are developed, deployed, and operated by Temporal customers.
+A _worker service_ is a service that hosts Workflow and Activity implementations.
+A worker service starts one or more Workers.
+A Worker polls a Task Queue for tasks, and executes Workflows in response to tasks.
+When you call `ExecuteWorkflow()`, Temporal adds a new task to the Task Queue, and a Worker executes the task.
 
-You can run a Temporal worker in a new or an existing service. Use the framework APIs to start the Temporal worker and link in all Activity and Workflow implementations that you require the service to execute.
+Worker services are developed, deployed, and operated by Temporal customers.
```

Comment:
i would use "Temporal users" rather than "customers" wdyt?

Reply:
Works for me, "customers" implies payment so I'm ok to make this change.